src/transformers/pipelines/text_to_audio.py

```diff
@@ -110,8 +108,10 @@ def _forward(self, model_inputs, **kwargs):
         # we expect some kwargs to be additional tensors which need to be on the right device
         kwargs = self._ensure_tensor_on_device(kwargs, device=self.device)
 
-        # call the generate by defaults or the forward method if the model cannot generate
-        output = self.forward_method(**model_inputs, **kwargs)
+        if self.model.can_generate():
+            output = self.model.generate(**model_inputs, **kwargs)
+        else:
+            output = self.model(**model_inputs, **kwargs)[0]
```

Comment:
If we generate, we return a torch tensor as our output.

If we do a forward pass, we return an instance of `ModelOutput`, where the 0-th element is our generated waveform

Hence, we slice here with `[0]` to get our waveform output